Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -256,6 +256,8 @@ extension SubscriptionsViewController {
         // some status changes
         if let subscription = ChatViewController.shared?.subscription {
             ChatViewController.shared?.chatTitleView?.subscription = subscription
+        } else {
+            ChatViewController.shared?.subscription = .initialSubscription()
```

Comment:
That could actually cause some problems because when we update the `.subscription` of `ChatViewController` we call a lot of other methods that aren't really needed all the time. That's why I'm only calling it when the `.subscription` is `nil`.